Sources/MountebankSwift/Models/Imposter/Imposter.swift

```diff
@@ -3,6 +3,43 @@ import Foundation
 /// Imposter as documented on:
 /// [mbtest.org/docs/api/contracts?type=imposter](https://www.mbtest.org/docs/api/contracts?type=imposter)
 public struct Imposter: Codable, Equatable {
+
+    /// Options for supported protocols by Mountebank.
+    public enum ExtraNetworkOptions: Codable, Equatable {
+        /// Options for the http protocol documented on:
+        /// [mbtest.org/docs/protocols/http](https://www.mbtest.org/docs/protocols/http)
+        /// - Parameters:
+        ///   - allowCORS: If true, mountebank will allow all CORS preflight requests on the imposter.
```

Comment:
"When true, mountebank will allow all Cross-Origin Resource Sharing preflight requests on the imposter" Perhaps write out the abbreviation just once?